.gitignore

```diff
@@ -3,3 +3,5 @@
 .idea
 node_modules
 .next
+public
+resources/_gen
```

Comment:
You can get rid of this line. This is from the old website that I still have to remove.

Reply:
done